generators/app/templates/src/index.js

```diff
@@ -1,7 +1,29 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { AppContainer } from 'react-hot-loader';
 
-import App from './App';
+import App from './app';
+import registerServiceWorker from './registerServiceWorker';
 import './config/i18n';
+import './scss/application.scss';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+const render = () => {
+  ReactDOM.render(
+    <AppContainer>
+      <App />
+    </AppContainer>,
+    document.getElementById('root')
+  );
+};
+
+// Render once
+render(App);
+
+registerServiceWorker();
+
+// Webpack Hot Module Replacement API
+if (module.hot) {
+  module.hot.accept('./app', () => {
+    render(App);
+  });
```

Comment:
I'd extract this configuration into another file. What do you think ?

Reply:
🤔 How?

Reply:
I thought something like:
```
function configureHotReloading(onConfigure) {
  module.hot.accept('~src/app', onConfigure);
}
```

Reply:
I'd also put here those steps that are needed to configure this module, like `render(App)` once from above.